quickstart/speech-translation/cpp-windows/helloworld/helloworld.cpp

```diff
@@ -0,0 +1,72 @@
+#include "pch.h"
+#include <iostream>
+#include <vector>
+#include <speechapi_cxx.h>
+
+using namespace std;
+using namespace Microsoft::CognitiveServices::Speech;
+using namespace Microsoft::CognitiveServices::Speech::Translation;
+
+// Translation with microphone input.
+void TranslationWithMicrophone()
+{
+	// Creates an instance of a speech translation config with specified subscription key and service region.
```

Comment:
>	 [](start = 0, length = 1)

no hard tabs; use 4 spaces for indentation (everywhere it applies)

Reply:
sorry for the white-space nit, but indentation is now inconsistent and only at two space ... can you have another look?

---
In reply to: [249393196](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/142#discussion_r249393196) [](ancestors = 249393196)